src/foremast/pipeline/construct_pipeline_block.py

```diff
@@ -137,6 +137,11 @@ def construct_pipeline_block(env='',
         data['asg'].update({'hc_type': 'EC2'})
         LOG.info('Switching health check type to: EC2')
 
+    # Aggregate the default grace period, plus the exposed app_grace_period
+    # to allow per repo extension of asg healthcheck grace period
+    hc_grace_period = data['asg'].get('hc_grace_period')
```

Comment:
That seems a bit too verbose for my tastes. How does it look now with separate variables?